source/ui/PGM/PGMBeamStatusView.cpp

```diff
@@ -0,0 +1,61 @@
+#include "PGMBeamStatusView.h"
+#include "beamline/PGM/PGMBeamline.h"
+
+#include <QGridLayout>
+#include <QLabel>
+#include <QGroupBox>
+
+
+PGMBeamStatusView::PGMBeamStatusView(QWidget *parent) :
+    QWidget(parent)
+{
+    ringCurrentLabel_ = new QLabel("Ring Current: ");
```

Comment:
Again, I'm wondering if we could do all this display with just a few generic widgets, rather than manually setting all the labels and values?